### ISSUE TITLE
Creating Unit Test for internal repositories

### ISSUE THREAD

**Issue by EgeAytin** [Permify/permify#85]
Writing unit tests for _internal/repositories/postgres_ and _internal/repositories/memory_  **relationshipReader**, **relationshipWriter**, **schemaReader**, **schemaWriter** repos.

**Comment by blackhorseya** [comment]
@EgeAytin Hi, Can you assign me on this?

**Comment by EgeAytin** [comment]
Hey @blackhorseya , any progress on this one? We're excited to see your changes here 👀 👀

**Comment by EgeAytin** [comment]
Hey @blackhorseya, any progress on this one? We're excited to see your changes here 👀 👀

**Comment by blackhorseya** [comment]
> Hey @blackhorseya, any progress on this one? We're excited to see your changes here 👀 👀

Sorry, I haven't started yet

**Comment by bhagatpratham** [comment]
Where can I find unit tests for repos?

**Comment by EgeAytin** [comment]
No unit test has been written yet for Repos @bhagatpratham

**Comment by bhagatpratham** [comment]
Can you help me then how to write one?

**Comment by EgeAytin** [comment]
@bhagatpratham Of course, we can discuss it on discord. Also we finished the [tests for relationshipReader](https://github.com/Permify/permify/blob/master/internal/repositories/postgres/relationshipReader_test.go) repository you can look at it as an example to start. If you're still interested I can assign you to this task ?
